packages/terms-of-service/tests/php/test_TermsOfService.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+
+namespace Automattic\Jetpack;
+
+use PHPUnit\Framework\TestCase;
+use phpmock\Mock;
+use phpmock\MockBuilder;
+
+class Test_Terms_Of_Service extends TestCase {
+
+	/**
+	 * Test setup.
+	 */
+	public function setUp() {
+		$this->terms_of_service = $this->createPartialMock( __NAMESPACE__ .'\\Terms_Of_Service',
+			array( 'get_raw_has_agreed', 'is_development_mode', 'is_active', 'set_agree', 'set_revoke' )
```

Comment:
Should we use the modern php syntax for declaring arrays, especially in tests?

Reply:
Done